__tests__/tests/actions/organization.action.js

```diff
@@ -0,0 +1,97 @@
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import {
+  GET_ORG_REPOS_LOADING,
+  GET_ORG_REPOS_ERROR,
+  GET_ORG_REPOS,
+} from '../../../src/organization/organization.constants';
+import { fetchOrganizationRepos } from '../../../src/organization/organization.action';
+import { v3 } from '../../../src/api';
```

Comment:
We have aliases configured. So you can write `api` directly.

Reply:
Nice. I'll change that. Tks!

Reply:
Done.